packages/prover/src/cli/cmds/start/options.ts

```diff
@@ -53,15 +53,15 @@ export const startOptions: CliCommandOptions<StartArgs> = {
   },
 
   beaconUrls: {
-    description: "The beacon node PRC urls for 'rest' mode.",
+    description: "The beacon node RPC urls for 'rest' mode.",
```

Comment:
The beacon node exposes a REST API and does not implement RPC, this is different from execution clients. 
The description also reference the mode 'rest' which doesn't even exist?

Reply:
RPC is a pretty generic term (unlink `JSON-RPC`), so I took it in this way. Can probably be just removed. 
Mention of `mode` looks like a left-over from previous refactoring.

Reply:
It even says that for the bootnodes for p2p which we don't even implement? Looks like it was just copy-pasted around without much thought

Reply:
I would suggest to remove the `beaconBootnodes` option for now in a following PR, as it's not supported (even so it shows a proper error message when used). Would help to clarify for users the available options.